### ISSUE TITLE
How to separete the dataset into testS and testR and the validation dataset in details?

### ISSUE THREAD

**Issue by wwxb2012** [Huang-ShiRui/Semi-UIR#22]
Can you provide the link for the training set and testing dataset?
As the author mentioned in the article,  UIEB dataset has 890 images, 800 are in the training dataset and 90 are in the testing dataset. However, according to the Supplementary Material, the validation set has 80 images from UIEB. Meanwhile, UWCNN dataset has 800 images in the training set, 110 images in the testing set, 120 images in the validation dataset while it contains 1449 images in total. It's confusing about how to divide the dataset.

**Comment by tengxiao14** [comment]
Do you know how to split the dataset? I have the same issue

**Comment by beimuyao** [comment]
I have the same issue. How did you solve it? And do you know what 'LA' means?

**Comment by beimuyao** [comment]
> Do you know how to split the dataset? I have the same issue
I have the same issue. How did you solve it? And do you know what 'LA' means? Thank you.
